grpc/handlers_backup.go

```diff
@@ -6,17 +6,28 @@ import (
 )
 
 func (srv *grpcServer) ToggleBucketBackup(ctx context.Context, request *pb.ToggleBucketBackupRequest) (*pb.ToggleBucketBackupResponse, error) {
-	bucketName := request.Bucket
+	bucketSlug := request.Bucket
 	bucketBackup := request.Backup
 
-	err := srv.sv.ToggleBucketBackup(ctx, bucketName, bucketBackup)
+	err := srv.sv.ToggleBucketBackup(ctx, bucketSlug, bucketBackup)
 	if err != nil {
 		return nil, err
 	}
 
 	return &pb.ToggleBucketBackupResponse{}, nil
 }
 
+func (srv *grpcServer) BucketBackupRestore(ctx context.Context, request *pb.BucketBackupRestoreRequest) (*pb.BucketBackupRestoreResponse, error) {
```

Comment:
I think this should be an internal call when starting up the daemon instead of being triggered by a client. That way we avoid adding a button to start the process when it should be handled in the background. What do you think?

Reply:
@dmerrill6 

Keeping both :-) as in https://github.com/FleekHQ/space-daemon/pull/207/files#diff-c253e3f5d0b08f2e201f5184bf4e886c6d878c93d8483a74550a5fae2e18b972R218